### PR TITLE
CI for Social Rearrange/Nav

### PR DESCRIPTION
**Pull request by ASzot**
## Motivation and Context

Added CI for social rearrange and social nav. Also updated the readme with the download instructions.

## How Has This Been Tested

With new test.

## Types of changes

- **\[Development\]** A pull request that add new features to the [habitat-lab](/habitat-lab) task and environment codebase. Development Pull Requests must be small (less that 500 lines of code change), have unit testing, very extensive documentation and examples. These are typically new tasks, environments, sensors, etc... The review process for these Pull Request is longer because these changes will be maintained by our core team of developers, so make sure your changes are easy to understand!

### CODE REVIEW

**Review by jimmytyyang — APPROVED** [review]
LGTM!

**Review by akshararai — APPROVED** [review]
Looks good to me, thanks for adding these!

**Review by xavierpuigf — APPROVED** [review]
LGTM!

File: habitat-baselines/README.md [hunk 1]
```diff
@@ -75,7 +75,8 @@ python -u -m habitat_baselines.run \
 
 To change the task (like set table) that you train your skills on, you can change the line `/habitat/task/rearrange: rearrange_easy` to `/habitat/task/rearrange: set_table` in the defaults of your config.
 
-# Multi-Agent
+# Habitat-3.0 Multi-Agent Training
+First download the necessary data with `python -m habitat_sim.utils.datasets_download --uids hssd-hab hab3-episodes habitat_humanoids hab3_bench_assets`.
```
Comment (akshararai) [inline habitat-baselines/README.md:79]: let's make sure that this command actually works by 5pm today.

Comment (akshararai) [reply habitat-baselines/README.md:79]: should probably also include ycb?

**Pull request merged**